gym/envs/tests/generate_json.py

```diff
@@ -0,0 +1,102 @@
+import gym
+from gym import envs, spaces 
+import json
+import numpy as np
+import datetime
+import os
+import hashlib
+
+import logging
+logger = logging.getLogger(__name__)
+
+from test_envs import should_skip_env_spec_for_tests
+
```

Comment:
Let's create a toplevel scripts/ folder in the gym repo and put this file there. That way it won't get shipped as part of the package distribution